backend/app/AppComponents.scala

```diff
@@ -229,7 +230,7 @@ class AppComponents(context: Context, config: Config)
 
       // external extractor
       val externalWorker = new ExternalTranscriptionWorker(manifest, sqsClient, config.transcribe, transcriptStorage, esResources)
-      val externalWorkerScheduler = new ExternalWorkerScheduler(actorSystem, externalWorker, config.worker.interval)(workerExecutionContext)
+      val externalWorkerScheduler = new ExternalWorkerScheduler(actorSystem, externalWorker, FiniteDuration.apply(5, TimeUnit.SECONDS))(workerExecutionContext)
```

Comment:
Maybe should move to config as well eventually? Unless it's just something which never varies between deployments, which it might be